src/index.js

```diff
@@ -190,6 +192,7 @@ export default new function () {
         if (error.statusCode === 401) {
           return this.auth.refreshTokens(performRequest, error);
         }
+        throw error;
```

Comment:
Resolves bug of Redux receiving a resolved promise (when it shouldn't) rather than the rejected promise